configs/mainnet.yaml

```diff
@@ -156,5 +159,6 @@ WHISK_EPOCHS_PER_SHUFFLING_PHASE: 256
 WHISK_PROPOSER_SELECTION_GAP: 2
 
 # EIP7594
-EIP7594_FORK_VERSION: 0x06000001
-EIP7594_FORK_EPOCH: 18446744073709551615
+NUMBER_OF_COLUMNS: 128
```

Comment:
As this is derived from other preset values, would it make more sense to keep it in the preset? (i.e. modifying this `NUMBER_OF_COLUMNS` in config would require a corresponding change on the `FIELD_ELEMENTS_PER_BLOB`/`FIELD_ELEMENTS_PER_CELL` values?

```
# `uint64((FIELD_ELEMENTS_PER_BLOB * 2) // FIELD_ELEMENTS_PER_CELL)` (= 128)
```

Reply:
I don't know if it should be a config value, but I think it can be. I think it's okay for the config values to be derived from the preset values, but not the other way around. Preset values shouldn't be derived from config values.